imagetest/utils/test_utils.go

```diff
@@ -14,7 +14,16 @@ import (
 	"cloud.google.com/go/storage"
 )
 
-const metadataURLPrefix = "http://metadata.google.internal/computeMetadata/v1/instance/"
+const (
+	metadataURLPrefix = "http://metadata.google.internal/computeMetadata/v1/instance/"
+)
+
+// OS Version Name
+const (
+	RedHat = "rhel"
+	Debian = "debian"
+	SUSE   = "suse"
```

Comment:
i don't understand the model you're going for, here. i think it's up to test author to figure out on what condition to test, right? and there are a lot more and a lot more sub-types than this.

Reply:
I thought these strings are something like fixed, so make it const.